service/javadsl/server/src/main/java/com/lightbend/lagom/javadsl/server/ServiceGuiceSupport.java

```diff
@@ -146,7 +146,7 @@ default void bindServices(ServiceBinding<?>... serviceBindings) {
      *
      * @return a {@link ServiceGuiceSupport.ServiceBinding} to be used as argument in {@link ServiceGuiceSupport#bindServices(ServiceBinding[])}.
      */
-    default <T extends Service> ServiceBinding<T> serviceBinding(Class<T> serviceInterface, Class<? extends T> serviceImplementation) {
+    default <T> ServiceBinding<T> serviceBinding(Class<T> serviceInterface, Class<? extends T> serviceImplementation) {
```

Comment:
In previous commits of this PR I change the capture type to a more strict (but more correct) and it breaks backwards compat (as detected by MiMa on Travis)